django/db/models/expressions.py

```diff
@@ -140,6 +140,12 @@ class BaseExpression:
     is_summary = False
     _output_field_resolved_to_none = False
 
+    # If the expression can be used in a WHERE-clause.
```

Comment:
"Can the expression be used in a WHERE-clause?"

Reply:
I think it was already commented a few times, so make yours final? :)